go.mod

```diff
@@ -34,7 +34,7 @@ require (
 require (
 	github.com/bgentry/go-netrc v0.0.0-20140422174119-9fd32a8b3d3d // indirect
 	github.com/cyberark/conjur-api-go v0.10.1 // version will be ignored by auto release process
-	github.com/cyberark/conjur-authn-k8s-client v0.23.8 // version will be ignored by auto release process
+	github.com/cyberark/conjur-authn-k8s-client v0.25.3-0.20230718150056-8ef20bd68e9d // version will be ignored by auto release process
```

Comment:
Not sure this matters but the relevant PR for authn-k8s-client bumped the version to v0.26.0, so it may be appropriate to bump it here as well when that release build is done

Reply:
Yes - I'll do this as soon as the release of authn-k8s-client is promoted